Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -400,9 +398,13 @@ namespace Remotion.BocAutoCompleteReferenceValue
                     }
 
                     if (event.keyCode == KEY.RETURN) {
-                        if (wasVisible) {
-                            // stop default to prevent a form submit, Opera needs special handling
-                            blockSubmit = true;
+                        if (options.isAutoPostBackEnabled) {
+                            // stop default on auto-postback since the change-event already takes care of this.
+                            event.preventDefault();
+                            return false;
+                        } else if (wasVisible) {
+                            // stop default for visible dropdown options since RETURN should simply select the current option without triggering a postback.
+                            event.preventDefault();
                             return false;
```

Comment:
Consider line 415 (ESC) also `event.preventDefault()` for `return false;`

Reply:
@a-ctor this discussion about preventDefault wasn't considered during the review. The latest code now uses the updated preventDefault semantsics.

Reply:
I don't know if we need `.preventDefault` here but it does not hurt. I don't think that the browser has a default behavior for ESC

Reply:
This is true, but given the original code defined it as `false`, the `preventDefault()` preserves the original semantics regardless of it being needed.